packages/parts/src/lib/maps/mapbox/map/Map.svelte

```diff
@@ -45,7 +46,14 @@
   const handlers: Record<string, any> = {
     dragend: () => dispatch('dragend', map.getCenter()),
     moveend: () => dispatch('moveend', map.getCenter()),
-    click: ({ lngLat }) => dispatch('click', lngLat),
+    click: (e) => {
+      if (
+        map
+          .queryRenderedFeatures(e.point)
+          .filter((f) => f.source.startsWith(ADDED_FEATURE_ID_PREFIX)).length === 0
```

Comment:
This is a quirk I solved: clicking on the region would not only activate it's popup but would signal a map click which in this case would start adding a secondary coordinate. So now when a map click contains a custom feature (like a region layer added by us) the main map click event won't fire.